### PR TITLE
Log message style changes

### PR DESCRIPTION
**Pull request by halter73**
- Increment connection id
- Disabled some log messages for possibly being too verbose.
- Don't log ConnectionReadFin when there is an error.

### CODE REVIEW

**Comment by halter73** [comment]
@drussilla Here are some additional logging changes we'd like to make.

**Comment by drussilla** [comment]
@halter73 Looks reasonable.

File: src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs [hunk 1]
```diff
@@ -21,57 +21,65 @@ public KestrelTrace(ILogger logger)
...
         {
-            _logger.LogDebug(10, $"{nameof(ConnectionDisconnect)} -> Id: {connectionId}");
+            _logger.LogDebug(10, @"Connection id ""{ConnectionId}"" disconnected.", connectionId);
         }
 
         public void ConnectionWrite(long connectionId, int count)
         {
-            _logger.LogDebug(11, $"{nameof(ConnectionWrite)} -> Id: {connectionId}, Count: {count}");
+            // Don't log for now since this could be *too* verbose.
+            //_logger.LogVerbose(11, @"Connection id ""{ConnectionId}"" sent ""{Count}"" bytes.", connectionId, count);
```
Comment (davidfowl) [inline src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs]: Don't leave in the commented out code. Just remove it and leave the comment.

Comment (halter73) [inline src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs]: @davidfowl This was to make it a little easier to do some debugging when working on Kestrel itself. I discussed this with @lodejard and he was fine with it. Do you feel strongly about this?

Comment (davidfowl) [inline src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs]: Yes. Dead code must die.

**Comment by halter73** [comment]
Thanks again @drussilla

**Pull request merged**